src/styles/blocks.scss/header-nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+    text-align: center;
+    list-style: none;
+    text-decoration: none;
```

Comment:
The `text-decoration: none;` style doesn't have any effect on the list items. It should be applied to the links instead. You already have it applied correctly in your `&__link` class.